### DISCUSSION TITLE
hashtags for reponsitories

### DISCUSSION THREAD

**Discussion by Variable-py** [community/community#29792] [New to GitHub] [answered]
Hi,
how can I set some hashtags for my reponsitory?
like this
![Likethem](https://user-images.githubusercontent.com/81676840/185106128-108eec9e-f980-4d2d-a0ec-09e033602dac.jpg)

**Comment by airtower-luna** [comment, marked as answer]
What do you mean by "hashtags"?

Git has *tags* (which mark specific commits within a repository), and GitHub has *topics* which are basically a list of keywords related to your repository.

**Reply by Variable-py** [reply]
I means them
![Likethem](https://user-images.githubusercontent.com/81676840/185106128-108eec9e-f980-4d2d-a0ec-09e033602dac.jpg)

**Reply by airtower-luna** [reply]
Those are topics, see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics#adding-topics-to-your-repository for how to set them. :slightly_smiling_face:

**Reply by Variable-py** [reply]
Thank you bro♥
